main.py

```diff
@@ -1,71 +1,45 @@
-import uuid
-
-from fastapi import Depends
-from fastapi import FastAPI, Form
-from fastapi import Request, Response
-from fastapi.responses import HTMLResponse
-from fastapi.templating import Jinja2Templates
-from sqlalchemy.orm import Session
+from sqlalchemy import Integer, String, Text
+from sqlalchemy.orm import Session, Mapped, mapped_column
 
 from database import Base
-from database import SessionLocal
-from database import engine
-from models import create_todo
```

Comment:
Good addition of the `notes` field. Consider adding a maximum length constraint if needed.